src/WattleScript.Interpreter/Execution/VM/Processor/Processor_InstructionLoop.cs

```diff
@@ -1788,8 +1795,12 @@ private int ExecIndex(Instruction i, int instructionPtr)
 
 			// stack: base - index
 			bool isNameIndex = i.OpCode == OpCode.IndexN;
+			bool isMultiIndex = i.OpCode == OpCode.IndexL;
 
-			bool isMultiIndex = (i.OpCode == OpCode.IndexL);
+			if (isNameIndex)
```

Comment:
Empty if statement?

Reply:
fixed in 277b828, used as a breakpoint, accidental commit